docs/1-Model-Levels-Walkthrough.ipynb

```diff
@@ -741,16 +751,19 @@
    },
    "source": [
     "## Model level wind-related variables (spherical harmonics)\n",
-    "- Divergence (1/s) (d)\n",
-    "- Vorticity (1/s) (vo)\n",
-    "- Temperature (K) (t)\n",
-    "- Vertical velocity (Pa/s) (w)\n",
+    "\n",
+    "The variables are listed in the same order as XArray displays them:\n",
```

Comment:
I like the function of this sentence to introduce that you're listing what the variables mean. However, I don't think so much attention needs to be focused on the variable order (that can be implied). Instead, I think this sentence should just state that the following are the variables in this Xarray Dataset.